FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,49 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+        client.get(from: url) { [weak self] result in
+            guard let self = self else { return }
+            self.mapCompletionResult(result, completion: completion)
```

Comment:
You can eliminate the `guard` with `self?` - the result is the same with one less line of code.

```suggestion
            self?.mapCompletionResult(result, completion: completion)
```